src/views/DetailView.js

```diff
@@ -18,6 +19,17 @@ export default class DetailView extends VivView {
           id: `${loader.type}${getVivId(id)}`,
           viewportId: id
         });
-    return [layer];
+    const { PhysicalSizeXUnit, PhysicalSizeX } = loader;
```

Comment:
I'd prefer that if these are on the loaders we use camelcase like other properties, rather than copying the omexml keys directly. Similar to `tileSize`, we assume that the `PhysicalSizeX` and `PhysicalSizeY` are the same for the image, so maybe we could just have the property `physicalSize` which is an object:

```javascript
const { unit, value } = loader.physicalSize;
```

or if we want to add all sizes and scale bar just assumes x:

```javascript
const { x, y, z } = loader.physicalSizes;
const { unit, value } = x;
```

Reply:
this will make it easy to implement on the zarr loader.

Reply:
You could even check size units in x and y and console log a warning if they are different.

Reply:
Thanks for this.  This was the thorniest point so I'm glad to have the feedback.